src/components/application_manager/include/application_manager/application_manager_impl.h

```diff
@@ -1445,6 +1475,14 @@ class ApplicationManagerImpl
    */
   protocol_handler::MajorProtocolVersion SupportedSDLVersion() const;
 
+  /**
+   * @brief Checks if driver distraction state is ON create message
+   * and put it to postponed message.
```

Comment:
@LitvinenkoIra does it out only driver distraction ON message to postponed? 
What this function does in case if distraction message is already in queue?